modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/logging/logmanager/LogManagerEncodingAttributeServerSetupTask.java

```diff
@@ -0,0 +1,68 @@
+package org.jboss.additional.testsuite.jdkall.present.logging.logmanager;
+
+import org.jboss.as.arquillian.api.ServerSetupTask;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.clustering.controller.Operations;
+import org.jboss.as.controller.client.helpers.ClientConstants;
+import org.jboss.as.test.integration.management.util.ServerReload;
+import org.jboss.as.test.integration.security.common.CoreUtils;
+import org.jboss.dmr.ModelNode;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.jboss.as.cli.Util.ALLOW_RESOURCE_SERVICE_RESTART;
+import static org.jboss.as.controller.client.helpers.ClientConstants.OPERATION_HEADERS;
+import static org.jboss.as.controller.client.helpers.ClientConstants.ROLLBACK_ON_RUNTIME_FAILURE;
+import static org.jboss.as.controller.client.helpers.ClientConstants.UNDEFINE_ATTRIBUTE_OPERATION;
+import static org.jboss.as.controller.client.helpers.ClientConstants.WRITE_ATTRIBUTE_OPERATION;
+import static org.jboss.as.test.integration.management.util.ModelUtil.createOpNode;
+
+/**
+ * Server setup task for test LogManagerEncodingAttributeTestCase.
+ * Sets "encoding" attribute on the file handler.
+ *
+ * @author Daniel Cihak
+ */
+public class LogManagerEncodingAttributeServerSetupTask implements ServerSetupTask {
```

Comment:
This needs the @EapAdditionalTestsuite annotation in order to be distributed to the specific server subsets.